src/app-layout/__integ__/app-layout-drawers.test.ts

```diff
@@ -69,160 +71,152 @@ interface SetupTestOptions {
+      await page.openThirdDrawer();
+      const originalWidth = await page.getActiveDrawerWidth();
+      await page.setWindowSize({ ...viewports.desktop, width: smallWindowWidth });
+      const newWidth = await page.getActiveDrawerWidth();
+      expect(newWidth).toBeLessThan(originalWidth);
+      expect(newWidth).toBeLessThan(smallWindowWidth);
+    })
+  );
+
+  testIf(theme !== 'visual-refresh-toolbar')(
+    `should not shrink drawer beyond min width`,
+    setupTest({ theme, screenSize: { ...viewports.desktop, width: 700 } }, async page => {
+      await page.openThirdDrawer();
+      await expect(page.getActiveDrawerWidth()).resolves.toEqual(290 + vrBorderOffset);
+    })
+  );
+
+  test(
+    'split panel and drawer can resize independently',
+    setupTest({ theme, splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1800 } }, async page => {
+      await page.openFirstDrawer();
+      await page.openSplitPanel();
+
+      const originalSplitPanelWidth = await page.getSplitPanelWidth();
+      const originalDrawerWidth = await page.getActiveDrawerWidth();
+      await page.dragAndDrop(wrapper.findSplitPanel().findSlider().toSelector(), 100);
+
+      const newSplitPanelWidth = await page.getSplitPanelWidth();
+      expect(newSplitPanelWidth).toBeLessThan(originalSplitPanelWidth);
+      await expect(page.getActiveDrawerWidth()).resolves.toEqual(originalDrawerWidth);
+
+      await page.dragAndDrop(wrapper.findActiveDrawerResizeHandle().toSelector(), -100);
+      await expect(page.getSplitPanelWidth()).resolves.toEqual(newSplitPanelWidth);
+      await expect(page.getActiveDrawerWidth()).resolves.toBeGreaterThan(originalDrawerWidth);
+    })
+  );
+
+  testIf(theme !== 'visual-refresh-toolbar')(
+    'updates side split panel position when using different width drawers',
+    setupTest({ theme, splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1450 } }, async page => {
+      await page.openFirstDrawer();
+      await page.openSplitPanel();
+      await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelSide().toSelector())).resolves.toEqual(true);
+
+      await page.openThirdDrawer();
+      await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelBottom().toSelector())).resolves.toEqual(true);
+
+      await page.openFirstDrawer();
+      await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelSide().toSelector())).resolves.toEqual(true);
+    })
+  );
+
+  test(
+    'scrolling drawer does not affect resize handle position',
+    setupTest({ theme }, async page => {
+      await page.openFirstDrawer();
+      const resizeHandleBefore = await page.getResizeHandlePosition();
+      await page.elementScrollTo(wrapper.findActiveDrawer().toSelector(), { top: 100 });
+      const resizeHandleAfter = await page.getResizeHandlePosition();
+      await expect(resizeHandleAfter).toEqual(resizeHandleBefore);
+    })
+  );
+
+  (theme === 'classic' ? test : test.skip)(
```

Comment:
only applicable to classic

Reply:
Should we use `testIf` here too for consistency?
```suggestion
  testIf(theme === 'classic')(
```